lib/class-wp-duotone-gutenberg.php

```diff
@@ -349,7 +358,15 @@ public static function render_duotone_support( $block_content, $block ) {
 		$filter_id = gutenberg_get_duotone_filter_id( array( 'slug' => $slug ) );
 
 		// Build the CSS selectors to which the filter will be applied.
-		$selector = WP_Theme_JSON_Gutenberg::scope_selector( '.' . $filter_id, $duotone_selector );
+		$selectors = explode( ',', $duotone_selector );
+
+		$selectors_scoped = array();
+		foreach ( $selectors as $selector_part ) {
+			// The selector part should always begin with the block's class, so we can safely just prepend the filter id class.
```

Comment:
Technically this would work with anything that isn't a type selector which includes tag names and namespaces. You can work around the tag names with the `:is()` pseudo-class, but namespaces are harder.